src/redux/data.js

```diff
@@ -614,7 +604,7 @@ const parseResponseBody = (response) => {
  * @param {Object} state - data store state leaf
  * @param {Object} action - redux action being dispatched
  */
-export function reducer(state = {}, action) {
+export function reducer(state, action) {
```

Comment:
Please, define initial value for the state `{}` inside function body

Reply:
@Yurii-Danylenko, the default value for state is already set and passed to this function (it's called `dataReducer`) [here](https://github.com/folio-org/ui-eholdings/blob/master/src/redux/index.js#L46).